src/browser/telemetry.js

```diff
@@ -1,15 +1,15 @@
 var _ = require('../utility');
 
 var defaults = {
-	network: true,
-	console: true,
-	dom: true,
-	location: true
+  network: true,
+  console: true,
+  dom: true,
+  location: true
```

Comment:
Oh, I can imagine we may want to allow users to capture gps location data but this key is for the nav location, correct? Maybe we should rename that to "navigation"?